content/en/deployment_gates/setup.md

```diff
@@ -239,8 +239,7 @@ spec:
 {{% /tab %}}
 {{% tab "Generic script" %}}
 
-
-Use this script as a starting point. For the API_URL variable, be sure to replace `<YOUR_DD_SITE>` with your [Datadog site name][1] (for example, {{< region-param key="dd_site" code="true" >}}).
+Use this script as a starting point. Be sure to replace `<YOUR_DD_SITE>` with your [Datadog site name][1] (for example, {{< region-param key="dd_site" code="true" >}}), `<YOUR_API_KEY>` with your [API key][2] and `<YOUR_APP_KEY>` with your [application key][3].
```

Comment:
List format is little cleaner for busy information like this:
```suggestion
Use this script as a starting point. Be sure to replace the following:

- `<YOUR_DD_SITE>`: Your [Datadog site name][1] (for example, {{< region-param key="dd_site" code="true" >}})
- `<YOUR_API_KEY>`: Your [API key][2]
- `<YOUR_APP_KEY>`: Your [application key][3]
```

How it renders:

<img width="600" height="344" alt="image" src="https://github.com/user-attachments/assets/0261aeda-aa11-4a02-b501-c0e76efa1c32" />